tests/slurm-tests/nano_30b_eval/run_test.py

```
@@ -0,0 +1,607 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""SLURM benchmark suite for NVIDIA-Nemotron-3-Nano-30B-A3B-BF16."""
+
+import argparse
+
+from nemo_skills.pipeline.cli import eval, prepare_data, run_cmd, wrap_arguments
```

Ruff (0.15.10)

[error] Import `eval` is shadowing a Python builtin

(A004)